observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+TLDR - Dashboard to show patterns. What you want to achieve here is to derive a pattern - user, usage, search or anything that you have defined as your KPI.
```

Comment:
Would be helpful to add more description around
* what is dashboard and visualization?
* why are these important to observability

Reply:
Using a dashboard to show/analyze patterns is only one use of a dashboard. There are other use cases such as health state or other "at a glance" information displays which should also be highlighted.

Reply:
added additional examples